modules/apps/captcha/captcha-test/src/testIntegration/java/com/liferay/captcha/internal/function/captcha/test/FunctionCaptchaImplTest.java

```diff
@@ -0,0 +1,137 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.captcha.internal.function.captcha.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.captcha.BaseCaptchaTestCase;
+import com.liferay.captcha.configuration.CaptchaConfiguration;
+import com.liferay.client.extension.type.CustomElementCET;
+import com.liferay.client.extension.type.configuration.CETConfiguration;
+import com.liferay.client.extension.type.manager.CETManager;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.configuration.metatype.bnd.util.ConfigurableUtil;
+import com.liferay.portal.configuration.test.util.CompanyConfigurationTemporarySwapper;
+import com.liferay.portal.configuration.test.util.ConfigurationTestUtil;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+import com.liferay.portal.kernel.util.HashMapDictionaryBuilder;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+
+import java.util.Map;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Pedro Victor Silvestre
+ */
+@RunWith(Arquillian.class)
+public class FunctionCaptchaImplTest extends BaseCaptchaTestCase {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new LiferayIntegrationTestRule();
+
+	@BeforeClass
+	public static void setUpClass() throws Exception {
+		_properties = HashMapBuilder.<String, Object>put(
+			"baseURL", "${portalURL}/o/test_" + _VIRTUAL_HOSTNAME
```

Comment:
Hey @daiazius, Can you please doublecheck if we need all these properties here? Please try to remove all that is not required. Thx